web/src/edit/NeighbourhoodMode.svelte

```diff
@@ -561,43 +562,53 @@
     </div>
 
     <h2>Map style</h2>
-
     <label>
       <input type="checkbox" bind:checked={$animateShortcuts} />
-      Animate shortcuts
+      Animate shortcuts<sup>[1]</sup>
+    </label>
+
+    <label>
+      <input type="checkbox" bind:checked={$drawBorderEntries} />
+      Show entries into cells<sup>[2]</sup>
+    </label>
+
+    <label>
+      <input type="checkbox" bind:checked={$thickRoadsForShortcuts} />
+      Road thickness depends on shortcuts<sup>[1]</sup>
+    </label>
+
+    <label style="display: flex; align-items: center; gap: 8px;">
+      <span style="text-wrap: nowrap;">Draw roads:</span>
+      <select
+        style="margin: 0; padding: 8px; width: auto;"
+        bind:value={$roadStyle}
+      >
+        <option value="shortcuts">Worst shortcuts</option>
+        <option value="cells">Cell</option>
+        <option value="edits">Edits (either filter or direction)</option>
+        <option value="speeds">Speed limit</option>
+      </select>
     </label>
+    {#if $roadStyle == "speeds"}
+      <SequentialLegend
+        colorScale={speedColorScale}
+        labels={{ limits: speedLimits }}
+      />
+    {/if}
+
+    <br />
     <p>
+      <sup>[1]</sup>
       <i>Shortcuts</i>
       are routes from one main road to another, which cut through the neighborhood's
       interior.
     </p>
-
-    <label>
-      <input type="checkbox" bind:checked={$drawBorderEntries} />
-      Show arrows into cells
-    </label>
     <p>
-      <i>Cells</i> show the area reachable without travelling along a main road.
+      <sup>[2]</sup>
```

Comment:
The footnotes take lots of space. I'm (slowly) investigating something like https://atomiks.github.io/tippyjs/ for NPW; tooltips might be a good approach here. The footnote notation is nice and recognizable for "there's more context here, maybe hover over?"

Reply:
Good idea! Added in #302